python/pylibvw.cc

```diff
@@ -712,30 +810,39 @@ void unsetup_example(vw_ptr vwP, example_ptr ae)
 
   if (all.add_constant)
   {
-    ae->feature_space[constant_namespace].clear();
-    int hit_constant = -1;
-    size_t N = ae->indices.size();
-    for (size_t i = 0; i < N; i++)
+    std::vector<unsigned char> bad_indices;
```

Comment:
All you need to do is remove the constant namespace. No other feature groups are affected

Reply:
All these changes are addressed in the latest commit